.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ['3.10']
```

Comment:
Why are we only testing Python 3.10 now? I'd like to keep some level of backwards compatibility.